obspy/clients/fdsn/client.py

```diff
@@ -1062,10 +1062,14 @@ def get_waveforms_bulk(self, bulk, quality=None, minimumlength=None,
             self._attach_dataselect_url_to_stream(st)
             return st
 
-    def get_stations_bulk(self, bulk, level=None, includerestricted=None,
-                          includeavailability=None, filename=None, format=None,
-                          **kwargs):
-        r"""
+    def get_stations_bulk(self, bulk,
+                     minlatitude=None, maxlatitude=None, minlongitude=None,
+                     maxlongitude=None, latitude=None, longitude=None,
+                     minradius=None, maxradius=None, level=None,
+                     includerestricted=None, includeavailability=None,
+                     updatedafter=None, matchtimeseries=None, filename=None,
+                     format=None, **kwargs)
```

Comment:
please add new kwargs after preexisting kwargs, to prevent breaking existing user code in case of using these kwargs as args (which isn't what you would usually do, but still)

Reply:
OK will do if you say so.. I did consider that initially but decided it may be better to make a small leap to maintain the same general structure as `get_stations` since this jumbled order with format and filename etc in the middle will be permanent after this.

Reply:
I agree, it might be slightly ugly, but I think being safe to not break code is more important.

I think we could consider to eventually rectify this by making a leap to introducing `, *,` in call syntax, making it `def get_stations_bulk(self, bulk, *, minlatitude=None, maxlatitude=None, ...)` to force using these kwargs by their name, which kind of is the only sane way to use them and what I would expect most people do anyway. When/if we do that as an announced potentially breaking change, we would then be free to reorder as we please.